plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/CoverageBuildAction.java

```diff
@@ -544,18 +544,29 @@ public String formatDelta(final Baseline baseline, final Metric metric) {
      * @param metric
      *         the metric to check
      *
-     * @return {@code true} if the trend is positive, {@code false} otherwise
+     * @return {@code 1} if the trend is positive, {@code -1} if the trend is negative, {@code 0} otherwise
      */
     @SuppressWarnings("unused") // Called by jelly view
-    public boolean isPositiveTrend(final Baseline baseline, final Metric metric) {
+    public int getTrend(final Baseline baseline, final Metric metric) {
         var delta = getDelta(baseline, metric);
         if (delta.isPresent()) {
-            if (delta.get().compareTo(Fraction.ZERO) > 0) {
-                return metric.getTendency() == MetricTendency.LARGER_IS_BETTER;
+            double deltaValue = delta.get().doubleValue();
+            // to apply rounding off
+            double roundedDelta = Math.round(deltaValue * 100.0) / 100.0;
```

Comment:
It would be much simpler to use:
If value < 0.1 and value > 0.1 then return 0. else return value. There’s no need to use -1 and +1, any value !=0 would make it.